lib/src/main/java/io/apptik/widget/MultiSlider.java

```diff
@@ -1111,8 +1115,10 @@ public boolean onTouchEvent(MotionEvent event) {
             //we normally have one single pointer here and its gone now
             case MotionEvent.ACTION_UP:
                 if (currThumb!=null) {
-                    setThumbValue(currThumb, getValue(event, currThumb), true);
+                    int value = getValue(event, currThumb);
+                    setThumbValue(currThumb, value, true);
                     onStopTrackingTouch();
+                    mOnThumbValueChangeListener.onStopTrackingTouch(this, currThumb, value);
```

Comment:
I will se if i can find some time during the week :)
Den 25/10/2015 10.20 PM skrev "Kalin Maldzhanski" <notifications@github.com
>:

> In lib/src/main/java/io/apptik/widget/MultiSlider.java
> <https://github.com/apptik/MultiSlider/pull/10#discussion_r42950559>:
>
> >                      onStopTrackingTouch();
> > +                    mOnThumbValueChangeListener.onStopTrackingTouch(this, currThumb, value);
>
> same as before, it is better to call in this inside onStopTrackingTouch()
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apptik/MultiSlider/pull/10/files#r42950559>.
>